src/gba/extra/proxy.c

```diff
@@ -282,7 +282,8 @@ void GBAVideoProxyRendererWriteVRAM(struct GBAVideoRenderer* renderer, uint32_t
 		proxyRenderer->backend->writeVRAM(proxyRenderer->backend, address);
 	}
 	if (renderer->cache) {
-		mCacheSetWriteVRAM(renderer->cache, address);
+		mCacheSetWriteVRAM(renderer->cache, address & ~1);
+		mCacheSetWriteVRAM(renderer->cache, address | 1);
```

Comment:
This is not the correct fix for this; for one, this function is called exactly the same way in multiple places, not just this function. For another, the API specifies what the width of some of these writes is supposed correlate with, but I'm not sure it's consistent so it might need to be reviewed in more depth.

Reply:
Solved by adding a map cache sysconfig bitfield member specifying whether a write updates only one, or two tiles. Please tell me if this approach is acceptable.

Reply:
I'm still not sure how I feel about this approach. The other fix looks fine now so maybe split it into two PRs so I can merge one first and then bikeshed the other one later.